users/kimurayoshimi/9.js

```diff
@@ -6,6 +6,6 @@ const c = (d) => d.some((value) => value === "e");
 const d = c(["a", "b", "c"]);
 console.log(d);
 
-const e = (f) => f.find((value) => value === "e");
+const e = (f) => f.every((value) => value === "e");
```

Comment:
## The commit was not corrected correctly, and has been fixed.
## Run-time commands
```
node ./users/kimurayoshimi/9.js
```
## [stackblitz](https://stackblitz.com/edit/node-qcr24v?file=users%2Fkimurayoshimi%2F9.js) d067e01